lib/project_types/theme/messages/messages.rb

```diff
@@ -290,6 +290,21 @@ module Messages
             Usage: {{command:%s theme list}}
           HELP
         },
+        share: {
+          help: <<~HELP,
+            {{command:%s theme share}}: Creates a shareable theme on your theme library.
+
+            Usage: {{command:%s theme share [ ROOT ]}}
```

Comment:
Should the usage mention that's an alias/equivalent to `push -u`?
Could avoid confusion around why these two look to be doing the same.